sources/packages/backend/workflow/src/workflow-definitions/load-assessment-consolidated-data.bpmn

```diff
@@ -111,6 +111,7 @@
           <zeebe:header key="studentDataAdditionalTransportPlacement" value="$.data.additionalTransportPlacement" />
           <zeebe:header key="applicationStatus" value="$.applicationStatus" />
           <zeebe:header key="applicationHasNOAApproval" value="$.hasNOAApproval" />
+          <zeebe:header key="appealsStudentDependantsData" value="$.appeals.studentDependantsAppeal.submittedData.dependants[*][&#39;dateOfBirth&#39;,&#39;declaredOnTaxes&#39;,&#39;attendingPostSecondarySchool&#39;]" />
```

Comment:
Also please map appealsStudentDependantsAppealData until `$.appeals.studentDependantsAppeal.submittedData` so that the parent question value `hasDependents` is also captured.

Reply:
Please ignore the previous comment and use either

1. updated xpath expression which can obtain hasDependants along with dependents array

or

2. use separate variable for `hasDependants` from appeal in load-assessment-consolidated data.